README.md

```diff
@@ -458,6 +464,10 @@ For information about the governance of the Node.js project, see
 * [ZYSzys](https://github.com/ZYSzys) -
 **Yongsheng Zhang** &lt;zyszys98@gmail.com&gt; (he/him)
 
+<details>
+
+<summary>Collaborator emeriti</summary>
+
 ### Collaborator emeriti
```

Comment:
I'm tempted to remove this now that it's in the `summary` tag, but doing so would probably break `node-core-utils` which parses the README file. In fact, the change as it stands here might already do that. (I don't think so, but worth checking.)

Reply:
Removing it would definitely break `find-inactive-collaborators.mjs` so maybe leave the removal for a subsequent PR that can fix that script and also test node-core-utils.

Reply:
A different option would be to change the summary to just say _Emeriti_ and have it be clear from context which it is, but then the header can be there in the `details` tag.  I'm rather liking that idea.